src/platform/Linux/ConnectivityManagerImpl.cpp

```diff
@@ -1223,7 +1223,13 @@ CHIP_ERROR ConnectivityManagerImpl::CommitConfig()
     gboolean result;
     std::unique_ptr<GError, GErrorDeleter> err;
 
-    ChipLogProgress(DeviceLayer, "wpa_supplicant: connected to network");
```

Comment:
we should keep this log to indicate if the network has been connected

Reply:
I've removed it because I thought that it was a typo. Are you sure that the `CommitConfig` is the right place for such message? Such message is here: https://github.com/project-chip/connectedhomeip/blob/956ead0324558d02fe5d6b55f460678080e00125/src/platform/Linux/ConnectivityManagerImpl.cpp#L1123

Reply:
Maybe log should say something like "ConnectivityManagerImpl::CommitConfig" since I do not acutally see wpa_supplicant specific things here. We will save config synchronously, so maybe even log "wpa_supplicant: will save config synchronously" or similar.

Reply:
> since I do not acutally see wpa_supplicant specific things here

Yeap, that was my reasoning behind removing this line. OK, I can replace this message with a more suitable one.

Reply:
I've added "wpa_supplicant: save config" log message (save config is wpa_supplicant specific).